### PR TITLE
tests: update documentation tests for callSpread contract migration

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
#agoric-sdk-branch: 1961-callSpreadPairs

### CODE REVIEW

**Review by katelynsills — CHANGES_REQUESTED** [review]
I think this doc needs the links updated as well https://agoric.com/documentation/zoe/guide/contracts/callSpread.html#issuers

We should probably add documentation for both examples, too.

**Comment by Chris-Hibbert** [comment]
I responded to your comments.

File: main/zoe/guide/contracts/pricedCallSpread.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+This contract implements a fully collateralized call spread. You can use a call spread as a
+[financial building block](https://youtu.be/m5Pf2d1tHCs) to create futures, puts, calls, and event
+binaries that would form the basis for a prediction market, insurance, and much more. A call spread
+is a combination of a call option bought at one strike price and a second call option sold at a
+higher price. A call spread has two participating seats that pay out complementary amounts depending
+on the value of some good at a known future time. This video gives a
+[walkthrough of the implementation](https://youtu.be/m5Pf2d1tHCs?t=3566) of the contract.
+
+There are two variants of the callSpread. This one has the creator specify the price that two
+parties will pay and give them each an invitation to buy a known position at a stated price.  The
```
Comment (katelynsills) [inline main/zoe/guide/contracts/pricedCallSpread.md:17]: This isn't the price and it isn't buying. It's that they are providing their own margin.

File: main/zoe/guide/contracts/pricedCallSpread.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+   stated time. After the deadline, it will issue a PriceQuote giving the value of the underlying
+   asset in the strike currency.
+
+<<< @/snippets/zoe/contracts/test-callSpread.js#startInstancePriced
+
+## Creating the Option Invitations
+
+The terms specify all the details of the options. A call to `creatorFacet.makeInvitationPair()` is
+required to specify the share (as a whole number percentage) that will be paid by the purchaser of
+the long position. It returns a pair of invitations.
```
Comment (katelynsills) [inline main/zoe/guide/contracts/pricedCallSpread.md:63]:
```suggestion
required to specify the share (as a whole number percentage) that will be paid by the holder of
the long position. It returns a pair of invitations.
```

File: main/zoe/guide/contracts/pricedCallSpread.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+<<< @/snippets/zoe/contracts/test-callSpread.js#makeInvitationPriced
+
+The creator gives these invitations to the two parties (or might retain one for their own use.) When
+Bob receives an invitation, he can extract the value of the call spread option that he wants to buy,
+and create a proposal. The collateral required is also in the option's details. The holders of the
```
Comment (katelynsills) [inline main/zoe/guide/contracts/pricedCallSpread.md:69]:
```suggestion
Bob receives an invitation, he can extract the value of the call spread option that he wants to receive,
and create a proposal. The collateral required is also in the option's details. The holders of the
```

**Review by katelynsills — CHANGES_REQUESTED** [review]
This seems correct apart from the mention of buying/purchaser/price, etc.

File: main/zoe/guide/contracts/README.md [hunk 4]
```diff
@@ -21,15 +21,23 @@ pre-built contracts that can be imported and run on Zoe:
...
+  separately.
+- [Loan Contract](./loan.md) - a basic collateralized loan contract.
 - [OTC Desk](./otc-desk.md) - A contract for giving quotes that can be
   exercised. The quotes are guaranteed to be exercisable because they
   are actually options with the underlying assets escrowed.
+- [Priced Call Spread](./pricedCallSpread.md) - Creates a pair of fully collateralized call spread
+  options. They are ERTP assets and can be used as such in other contracts. There are two variants
+  of this contract, which affect the way invitations are created. In this version, the creator
+  requests a pair of options, each of which enables the holder to purchase a particular position at
+  a stated price. This version would be useful for a market maker who finds pairs of people with
```
[the remark this answers is not in the document]
Comment (Chris-Hibbert) [reply main/zoe/guide/contracts/README.md]: changed

File: main/zoe/guide/contracts/fundedCallSpread.md [hunk 5]
```diff
@@ -1,22 +1,26 @@
...
+[walkthrough of the implementation](https://youtu.be/m5Pf2d1tHCs?t=3566) of the contract.
 
-The Zoe invitations representing the options are produced in
-pairs.  The creator deposits the entire amount that will be apportioned between the holders of
-the two positions. The individual options are ZOE invitations whose details are inspectable by
-prospective purchasers.
+There are two variants of the callSpread. This one is fully funded by its creator, who can then sell
+(or transfer another way) the options to other parties. The other is called the
+[pricedCallSpread](./pricedCallSpread.md). It allow the creator to specify the price that two
+parties will pay, and give them each an invitation to buy a known position at a stated price.
```
[the remark this answers is not in the document]
Comment (Chris-Hibbert) [reply main/zoe/guide/contracts/fundedCallSpread.md]: updated

File: main/zoe/guide/contracts/pricedCallSpread.md (hunk 1 above)
Comment (Chris-Hibbert) [reply main/zoe/guide/contracts/pricedCallSpread.md]: updated

File: main/zoe/guide/contracts/pricedCallSpread.md (hunk 2 above)
Comment (Chris-Hibbert) [reply main/zoe/guide/contracts/pricedCallSpread.md]: done

File: main/zoe/guide/contracts/pricedCallSpread.md (hunk 3 above)
Comment (Chris-Hibbert) [reply main/zoe/guide/contracts/pricedCallSpread.md]: done

**Review by katelynsills — APPROVED** [review]
LGTM!

**Pull request merged**